Rakefile

```diff
@@ -17,7 +17,7 @@ task :default => [:test]
 
 desc 'Run RSpec'
 RSpec::Core::RakeTask.new(:test) do |t|
-  t.pattern = 'spec/{unit}/**/*.rb'
+  t.pattern = 'spec/{classes,defines,unit,functions,hosts,integration}/**/*_spec.rb'
```

Comment:
@glennsarti that's a great catch. I assumed that JJB was doing that, but looking closer at the spec runs it appears their restricted directory match was masking the fact they don't actually do the prep. This needs to be updated so that rake test => rake spec_prep